apps/editing-toolkit/bin/sync-newspack-blocks.sh

```diff
@@ -134,7 +134,6 @@ echo -n "phpcbf: "
 
 if [ "$PHPCBF_ERRORED" = 1 ] ; then
 	echo '!! There was an error executing phpcbf!'
-	exit 1
```

Comment:
> What do you think about keeping the error and throwing it the build process (presumably used for production build), but not for the dev build

This makes total sense to me, but I had assumed this was only used for the dev build ( might totally be mistaken ). If this is, in fact, used for prod, then I agree

Reply:
@noahtallen will know better, but I don't think this is (or should) be used for production releases. We use the `install-script.sh` from the sandbox, and the assets are fetched directly from the TC build associated with the PR. Now, nothing prevents you from building a production release and syncing, so it might be useful for releasing to production in some edge cases?

Reply:
If we want to be careful and still allow `phpcbf` to run when building for production, we can just test against `$NODE_ENV` like this:

```
if [ "$PHPCBF_ERRORED" = 1 ] && [ "$NODE_ENV" = "production" ] ; then                                                                                                                                           
   echo '!! There was an error executing phpcbf!'                                                                                                                                                                
   exit 1                                                                                                                                                                                                        
fi
```

Reply:
There's a `release` mode referenced in the file, so I was inferring that this is used to make a production build of the blocks for Newspack sites (perhaps the file is poorly named?).

At least, I assume these blocks are still distributed separately for Newspack sites (in addition to being including in ETK), but maybe someone from that team can verify. @Automattic/newspack

Reply:
Ah good point, even though it's part of ETK, we're specifically talking about the newspack blocks syncing script, which seems to be part of a slightly different (and separate) release flow.

Reply:
Hmm thanks for digging in further, I naively thought this was just for sandbox sync and could be safely skipped.

We should probably just try to skip any/all functionality that isn't related to `yarn dev --sync` in this Newspack script and/or split out the concerns into separate scripts.

Reply:
[It sounds like this is a dev script](https://github.com/Automattic/wp-calypso/pull/65043#issuecomment-1170337744).

To be extra safe though, I added an extra condition. I verified that we are in `MODE='npm'`  when running `yarn dev --sync` and _only_ skip the exiting logic in that case.  https://github.com/Automattic/wp-calypso/blob/89b10bccbb313c441e60d5ce0a7ec59dbba741d7/apps/editing-toolkit/package.json#L26 How does that sound to y'all for the time being?

Reply:
Sorry for the late reply. Looks like I was mistaken and this isn't used by Newspack at all, so we're free to edit this as needed. In that case removing the exit line altogether seems fine, but I'm also good with the conditional check that you ended up with.

Thanks for landing this!

Reply:
Update... it looks like throwing an error for ill-formatted code in the production build is still desired: paYE8P-1I4-p2#comment-1588